package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "surveyor",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --outputFile test-results.json --json",
+    "eject": "react-scripts eject",
+    "format": "prettier \"src/**/*.{js,html}\" --write",
+    "lint": "eslint \"src/**/*.{js,jsx}\" --quiet",
+    "stylelint": "stylelint '**/*.scss'",
+    "lint-in-file": "eslint \"src/**/*.{js,jsx}\" --quiet --output-file ./eslint-output.json -f json",
+    "danger": "npx danger ci"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "dependencies": {
+    "node-sass": "4.14.1",
+    "react": "16.13.1",
+    "react-dom": "16.13.1",
+    "react-scripts": "3.4.3"
+  },
+  "devDependencies": {
+    "@testing-library/jest-dom": "4.2.4",
+    "@testing-library/react": "9.5.0",
+    "@testing-library/user-event": "7.2.1",
+    "babel-eslint": "10.1.0",
+    "cross-env": "7.0.2",
+    "danger": "10.3.0",
+    "danger-plugin-jest": "^1.3.0",
```

Comment:
@rafayet-monon At this block also 🙌
We can remove `^` out.

Reply:
Fixed 🚀